tasks/sql-connector/task.yaml

```diff
@@ -0,0 +1,37 @@
+id: sql-connector
+name: SQL Connector
```

Comment:
This should be called `transfer-data-from-sql-database`, or `upload-data-from-sql-database`. Also, `id` and `name` must match, we will be removing `name` completely soon, so keep them identical until we do.